apps/todomvc-remix/app/routes/api.auth.callback.google.tsx

```diff
@@ -0,0 +1,61 @@
+import { LoaderFunction, redirect } from "@remix-run/node";
+import { keelAccessCookie, keelRefreshCookie } from "~/cookies.server";
+
+const clientId = process.env.GOOGLE_CLIENT_ID;
+const clientSecret = process.env.GOOGLE_CLIENT_SECRET;
+
+if (!clientId) {
+  throw new Error("Missing GOOGLE_CLIENT_ID");
+}
+
+if (!clientSecret) {
```

Comment:
Does this file/code not get executed in the browser? You wouldn't want the client secret making it into browser code.

Reply:
The `loader` function does not get executed in the browser. :D

Reply:
No but this code is in module scope, not in the loader. It might be a non-issue anyway as this route is basically server-only as it does a redirect. If it works then 👍